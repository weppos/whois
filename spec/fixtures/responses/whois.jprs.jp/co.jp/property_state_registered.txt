[ JPRS database provides information on network administration. Its use is    ]
[ restricted to network administration purposes. For further information,     ]
[ use 'whois -h whois.jprs.jp help'. To suppress Japanese output, add'/e'     ]
[ at the end of command, e.g. 'whois -h whois.jprs.jp xxx/e'.                 ]

Domain Information:
a. [Domain Name]                YAHOO.CO.JP
g. [Organization]               Yahoo Japan Corporation
l. [Organization Type]          Corporation
m. [Administrative Contact]     
n. [Technical Contact]          
p. [Name Server]                ns01.yahoo.co.jp
p. [Name Server]                ns02.yahoo.co.jp
p. [Name Server]                ns11.yahoo.co.jp
p. [Name Server]                ns12.yahoo.co.jp
s. [Signing Key]                
[State]                         Connected (2015/11/30)
[Registered Date]               2000/11/17
[Connected Date]                2000/11/17
[Last Update]                   2015/02/27 10:26:20 (JST)

