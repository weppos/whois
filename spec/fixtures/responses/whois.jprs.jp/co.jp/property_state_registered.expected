#disclaimer
  %s == "JPRS database provides information on network administration. Its use isrestricted to network administration purposes. For further information,use 'whois -h whois.jprs.jp help'. To suppress Japanese output, add'/e'at the end of command, e.g. 'whois -h whois.jprs.jp xxx/e'."

#registered?
  %s == true

#domain
  %s == "YAHOO.CO.JP"

#status
  %s == :registered

#available?
  %s == false

#created_on
  %s == Time.zone.parse("2000/11/17 JST")

#updated_on
  %s == Time.zone.parse("2015-02-27 10:26:20 JST")

#registrar
  %s be_a(Whois::Record::Registrar)
  %s.organization == "Yahoo Japan Corporation"
